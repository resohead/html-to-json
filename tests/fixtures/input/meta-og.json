[
    { "charset": "utf-8" },
    { "property": "og:title", "content": "Example Site Title Meta" },
    { "property": "og:type", "content": "article" },
    { "property": "og:url", "content": "https://example.com" },
    {
        "property": "og:image",
        "content": "https://example.com/social-image-1.jpg"
    },
    {
        "property": "og:image",
        "content": "https://example.com/social-image-2.jpg"
    },
    {
        "property": "og:description",
        "content": "This is a test og description tag"
    },
    { "property": "og:site_name", "content": "Example Site Name Meta" }
]
