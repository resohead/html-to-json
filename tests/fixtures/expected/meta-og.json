{
    "hybrid": {
        "url": "https://example.com",
        "handle": null,
        "title": "Example Site Title Meta",
        "name": "Example Site Name Meta",
        "description": "This is a test og description tag",
        "type": "article",
        "images": [
            "https://example.com/social-image-1.jpg",
            "https://example.com/social-image-2.jpg"
        ],
        "image": "https://example.com/social-image-2.jpg",
        "author": null,
        "authorHandle": null,
        "keywords": null
    },
    "openGraph": {
        "images": [
            "https://example.com/social-image-1.jpg",
            "https://example.com/social-image-2.jpg"
        ],
        "title": "Example Site Title Meta",
        "type": "article",
        "url": "https://example.com",
        "image": "https://example.com/social-image-2.jpg",
        "description": "This is a test og description tag",
        "siteName": "Example Site Name Meta"
    },
    "twitter": {
        "images": []
    }
}
